libs/ui/src/signed_hash_validation_button.tsx

```diff
@@ -109,9 +119,17 @@ function SignedHashValidationModal({
               <Caption>
                 <strong>Machine ID:</strong> {machineId}
               </Caption>
-              <Caption>
-                <strong>Election ID:</strong> {combinedElectionHash || 'None'}
-              </Caption>
+              {!hideElectionId && (
+                <Caption>
+                  <strong>Election ID:</strong> {combinedElectionHash || 'None'}
+                </Caption>
+              )}
+              {showPollbookPackageId && (
+                <Caption>
+                  <strong>Pollbook Package ID:</strong>{' '}
```

Comment:
On a lesser note, should this read "Poll Book Package ID"? Since the agreed upon canonical product name is VxPollBook and product category is a poll book? (We don't need to update all variable names, just going for consistency in all external presentations)